### PR TITLE
Fixes arrow buttons and slider animations

### PR DESCRIPTION
**Pull request by cdata**
- Replace placeholder arrow buttons with paper-icon-button
 - Fix issue coordinating iron-selectable selection / deselection that
   caused slider to not animate.
 - Misc. doc cleanup.

Addresses #13 and #16.

/cc @morethanreal @notwaldorf

### CODE REVIEW

File: paper-tabs.html [hunk 1]
```diff
@@ -215,28 +207,22 @@
 
       /**
        * If true, the bottom bar to indicate the selected tab will not be shown.
-       *
-       * @attribute nobar
        */
-      nobar: {
+      noBar: {
```
Comment (morethanreal) [inline paper-tabs.html]: Just a bikeshed and you can ignore, but why `noink` but `noBar`?

**Comment by morethanreal** [comment]
Might it be possible to add a test for the animation issue?

**Comment by morethanreal** [comment]
and lgtm

File: paper-tabs.html (hunk 1 above)
Comment (cdata) [inline paper-tabs.html]: I guess my change is a bikeshed too. `no` and `bar` are two words, and I think that the attribute is better expressed in camel case. `noink` is used in a lot of places, so even though I would rather change it, I wasn't sure if I should. What do you think? I'm happy to revert if you think this is too much bike shedding.

**Comment by frankiefu** [comment]
lgtm

**Pull request merged**